### ISSUE TITLE
Network Error - 99 - Error creating SSL context ()

### ISSUE THREAD

**Issue by themaneiac** [Sibusten/derpibooru-downloader#13]
This error has been happening for at least a few days now: 
![image](https://user-images.githubusercontent.com/6430050/47960999-17c06600-dfc0-11e8-9498-9092f98a9386.png)
I didn't say anything because I thought somebody else would have by now. It still does it if I try re-downloading the downloader.

**Comment by themaneiac** [comment]
Okay, this is weird.. After days of trying it and it not working I tried it again just now and it works finally. I didn't do anything differently. When I upgraded to 1.4.0 when it came out I removed everything except the images and json folders, moved the new version in, then I ran the program and closed it then I added the "DerpibooruDownloader.ini" back in case the ini file being there would have stopped it from generating needed files or something. I just restarted the computer for the first time in a couple weeks so restarting might have fixed it. I had BitLocker encrypting 12 TB of files so I couldn't restart the computer before that :/
Sorry for wasting your time lol

I do have a quick question.. Does this program detect not-yet-rendered files that just got uploaded? I have a thing that downloads all the images as they come out and a lot of files turn out like this instead of what the image is supposed to be: 
![sample](https://user-images.githubusercontent.com/6430050/47997528-05722500-e0b1-11e8-9d92-d8869e528c45.jpg)

Thank you!
